retriever/lib/compile.py

```diff
@@ -162,38 +159,43 @@ def compile_json(json_file):
 
                     elif t_key == "schema":
                         add_schema(table_dict, table)
+                    else:
+                        table_dict[t_key] = t_val
 
                 tables[table["name"]] = table_dict
-
+            values["tables"] = tables
         else:
             values[key] = value
 
-    # Create a Table object string using the tables dict
-    table_desc = "{"
-    for (key, value) in tables.items():
-        table_desc += "'" + key + "': Table('" + key + "', "
-        table_desc += ','.join([key + "=" + str(value)
-                                for key, value, in value.items()])
-        table_desc += "),"
-    if table_desc != '{':
-        table_desc = table_desc[:-1]
-    table_desc += "}"
-
-    values["tables"] = table_desc
-
-    script_desc = []
-    for key, value in values.items():
-        if key not in keys_to_ignore:
-            script_desc.append(key + "=" + str(value))
-    script_desc = (',\n' + ' ' * 27).join(script_desc)
-
-    if 'template' in values.keys():
-        template = values["template"]
-    else:
-        template = "default"
-    script_contents = (script_templates[template] % script_desc)
-
-    new_script = open(json_file + '.py', 'w', encoding='utf-8')
-    new_script.write('# -*- coding: {}  -*-\n'.format(source_encoding.lower()))
-    new_script.write(script_contents)
-    new_script.close()
+    # # Create a Table object string using the tables dict
```

Comment:
So the Values contains all the values that we want to initialize in an script class